README.md

```diff
@@ -119,12 +120,28 @@ An example structure could look like this:
 ```
 
 If the locale that the user currently is using doesn't have a corresponding html file it will fall back to
-`slide_en_US.html`.
+`slide_en_US.html`. See the [Language code format section](#language-code-format) for further details about the language
+code format.
 
 Do note that the HTML supported in these “html” files is far from the full standard, so we recommend that you use one of
 the templates provided as the default slides:
 https://github.com/canonical/ubuntu-desktop-provision/tree/main/packages/ubuntu_bootstrap/assets/slides
 
+### EULA
+
+When `mode: oem` is set, the stage 4 eula page is activated. EULA assets are expected in
```

Comment:
Currently ubuntu_init completely ignores the `mode` setting and behaves according to the `eula` case by default. We should probably implement a custom page loading mechanism for ubuntu_init similar to `InstallerService.load()`.

Reply:
I'll just remove the first sentence for now then